ProjectLibertyLabs/gateway
# Doc Review Updates

Fixes to the docs, both in the markdown and in the swagger and comments

- Grammar fixes
- Wording fixes
- OpenAPI consistency
- "Sign In With Frequency" (no dash, etc..)
- "MSA Id" in text `msaId` or `xMsaId in code
- etc...

NOT in this PR: Removing `DTO/Dto` from names

## Steps to Verify:

- `make format`, `make generate-openapi` should have no changes remaining
- `cd docs && mdbook serve`
